examples/memory/demo_memory_pinecone.py

```diff
@@ -1,19 +1,18 @@
-import os
-
 from dynamiq.components.embedders.openai import OpenAIEmbedder
 from dynamiq.connections import Pinecone as PineconeConnection
 from dynamiq.memory import Memory
 from dynamiq.memory.backend import Pinecone
 from dynamiq.prompts import MessageRole
 
+
 pinecone_connection = PineconeConnection()
 embedder = OpenAIEmbedder(dimensions=1536)
+
```

Comment:
This works just if the default index already exists.

Reply:
Based on your previous comment, if there is an error, we will set up 'index_type.' What do you suggest?